docs/ref/models/querysets.txt

```diff
@@ -142,6 +142,11 @@ Here's the formal declaration of a ``QuerySet``:
         :meth:`order_by()` clause or a default ordering on the model.
         ``False`` otherwise.
 
+    .. attribute:: empty
+
+        ``True`` if the ``QuerySet`` is an instance of ``EmptyQuerySet``,
+        e.g. ``Article.objects.none()``
```

Comment:
I think it's more accurate to say:

```
``True`` if the query is guaranteed to return no results, because it is an ``EmptyQuerySet`` created with ``none()``.
```

I can imagine this property being used for more than `EmptyQuerySet` in the future as an optimization, e.g. for a queryset created with an empty slice.